rpc/core/pipe.go

```diff
@@ -44,7 +44,8 @@ type transport interface {
 }
 
 type peers interface {
-	DialPeersAsync(p2p.AddrBook, []string, bool) error
+	AddPersistentPeers([]string) []error
+	DialPeersAsync([]string) []error
```

Comment:
Similarly, `DialPeersAsync`: if the errors are not ignored (which is well explained tho), only the first error is considered.